src/main/resources/templates/hello.html

```diff
@@ -9,55 +9,30 @@
 
 <h1>Hello! Let's search an airport name by 3-letter code!</h1>
 <br>
-<form method="post" action="/database/airport_database" name="writeDownForm">
-    Airport code:<input type="text" name="airport_code" th:value="${airport_code_value}"/>
-    <input type="submit" value="Click here"/>
+
+<form method="post" action="/database/airport_database" name="pullDownForm">
+    Airport code:
+    <select id="selectedCode" name="codeToSelect">
+        <option value="">Select code</option>
+        <option th:each="airportCode : ${airportCodeList}"
+                th:value="${airportCode}"
+                th:text="${airportCode}"
+                th:selected="${airportCode == selectedValue}">
+        </option>
+    </select>
+
+    <button type="submit" id="CodeToBeInput" name="airport_code" onclick="updateValue()">Search</button>
```

Comment:
idのケースは揃えましょう。
キャメルケースで良いかと思います。
```suggestion
    <button type="submit" id="codeToBeInput" name="airport_code" onclick="updateValue()">Search</button>
```

Reply:
ご指摘箇所を修正しました。
4ff13c5487d8ab6888654ab5e38884a78e622044